### PR TITLE
update bundle to be compatible with Symfony 3.0

### PR DESCRIPTION
**Pull request by samdjstevens**
PR to make the bundle installable with Symfony 3.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.1",
-        "symfony/security-bundle": "~2.1",
+        "php": ">=5.5.9",
+        "symfony/framework-bundle": "~2.1|^3.0",
+        "symfony/security-bundle": "~2.1|^3.0",
```
Comment (lennerd) [inline composer.json]: Please remove the old versions of Symfony.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+matrix:
+  allow_failures:
+    - php: 7
+  include:
+    - php: 5.5.9
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 7
+      env: SYMFONY_VERSION=3.0.*@dev
```
Comment (lennerd) [inline .travis.yml]: As you use the global env `SYMFONY_VERSION=3.0.*@dev` it's redudant to include them in the matrix.

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,27 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
+  - 5.5.9
   - 5.6
```
Comment (lennerd) [inline .travis.yml]: Add PHP version 7.

**Comment by lennerd** [comment]
Thanks for your PR! 👍  See my line notes.

**Comment by samdjstevens** [comment]
Thanks for the pointers - I'm quite new to Travis CI!

**Comment by lennerd** [comment]
No problem. It's allways the same for me. Allways need to take a look into the documentation when changing the configuration. 😉 

Everything looks fine now. Can you just run `composer update` again to update the `composer.lock` file?

File: .travis.yml [hunk 4]
```diff
@@ -1,14 +1,25 @@
...
+env:
+  - SYMFONY_VERSION=3.0.*@dev
+
+matrix:
+  allow_failures:
+    - php: 7
+  include:
+    - php: 5.5.9
+    - php: 5.6
+    - php: 7
```
Comment (lennerd) [inline .travis.yml]: These include block can be removed completly. It's only used to overwrite global settings like for example the env variables.

**Comment by samdjstevens** [comment]
Sure, will run composer update now and remove the entire include block. Do you want all the commits squashed into a single one?

**Comment by lennerd** [comment]
Squashing them would be awesome.

**Comment by lennerd** [comment]
I'm totally sorry but I know saw that I did a mistake when pointing you to the Travis configuration file. As we will remove the old version of Symfony from this bundle, we can simply use the old configuraiton file, which uses the Symfony version from the composer.lock file.

Adding configuration for PHP 7 to the `travis.yml` should be enough. So please remove the configuration for the env variable and the line it's get pased down to the composer install command. (https://github.com/lennerd/VipxBotDetectBundle/pull/14/files#diff-354f30a63fb0907d4ad57269548329e3R18).

🙈

**Comment by samdjstevens** [comment]
I'll get there eventually, haha. No worries, I'll modify the `.travis.yml` file and take a look at the test failures later on tonight. Thanks for the help so far.

**Comment by samdjstevens** [comment]
Let me know if this looks good to you and I'll squash them together again.

**Comment by lennerd** [comment]
Great work also fixing the failed test. Looks good to me. Let me know when you squashed everything, so I can release version 3 of this bundle. 😉

**Comment by samdjstevens** [comment]
All done! Thanks for your help with it. Will be looking out for the release 👍

**Comment by lennerd** [comment]
Thank you! 👍

**Pull request merged**